integrationtests/test_all_examples.py

```diff
@@ -279,6 +281,24 @@ def run_file(file_path):
         exec(content, globals)
 
 
+def load_demo(file_path, variable_name="demo"):
+    """ Loads a demo example from given file_path. Extracts the relevant
+    object via variable_name.
+    Parameters
+    ----------
+    file_path : str
+        The file_path of the file to be loaded
+    variable_name : str
+        The key in the global symbol state corresponding to the object of
+        interest for the demo.
```

Comment:
This function returns something, please add a "Returns" block here.  We expect the return object to be an instance of `HasTraits` because we will need to call `edit_traits` on it. That should be mentioned.